### ISSUE TITLE
unable to use recently (today 3/29/23)

### ISSUE THREAD

**Issue by Urim512** [joshhunt/GhostOverlay#51]
i have issue recently i tried to use it i cant add or remove anything any time i try it ends up crashing says something isn't playing nice and i either have to have it retry and it keeps crashing everytime i restart it on my pc which is all i have please help fyi i have language set to English US if you need to know

![Screenshot 2023-03-29 133019](https://user-images.githubusercontent.com/129323260/228634553-b559761d-a2fd-49be-918f-74ad70635f41.png)
 
 to summarize everytime i click this button it opens the window but crashes shortly after please help me

i really would like to use the part for crucible to help me

**Comment by arcomador** [comment]
Same as issue 47, Dev has pushed update to MSFT store for approval, all that's left is the waiting.
#47
